css/components/features.css

```diff
@@ -0,0 +1,124 @@
+h2 {
+    font-family: var(--font-family-primary);
+    font-size: 46px;
+    font-weight: 600;
+    margin-top: 60px;
+    margin-bottom: 60px;
+    line-height: 1.25;
+    text-align: center;
+}
+
+.features {
+    position: relative;
+    width: 1380px;
+    margin: auto;
+    margin-top: 100px; 
+    margin-bottom: 100px; 
+}
+
+.features > .features-boxes {
+    background-color: #4e2fa9;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    justify-content: center;
+    text-align: center;
+    row-gap: 70px;
+    column-gap: 40px;
+    padding-top: 80px;
+    padding-bottom: 80px;
+    border-radius: 20px;
+}
+
+.features h3 {
+    font-family: var(--font-family-primary);
+    font-weight: 500;
+    font-size: 26px;
+    margin-top: 28px;
+    color: white;
+}
+.features-box1 {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+}
+.features-box1 > i {
+    width: 120px;
+    height: 120px;
+    background-color: white;
+    border-radius: 10px;
+    background-image: url("/svg/pf1.svg");
```

Comment:
manau is features-box(1-4) bei features box(1-4) > i  galetum padaryti 2 aprasymus (Nes pas tave viskas kartojasi, bet tik background-image keiciasi ) taip sutrumpinant koda. taip padaryti gali duodant <i> tagui klase su svg paveiksliuku, o features-box pavadinant visus vienodais klasiu vardais